hvpy/utils.py

```diff
@@ -49,6 +49,28 @@ def convert_date_to_unix(v: list) -> str:
     return ",".join([str(int(datetime.timestamp(d))) for d in v])
 
 
+def _data_source_to_int(source: Union[int, DataSource]) -> int:
+    """
+    Converts a `~hvpy.DataSource` to an integer.
+
+    Parameters
+    ----------
+    source
+        The `~hvpy.DataSource` to convert.
+
+    Returns
+    -------
+    int
+        The integer representation of the `~hvpy.DataSource`.
+    """
+    if isinstance(source, DataSource):
+        return source.value
+    elif isinstance(source, int) and source in [x.value for x in DataSource]:
+        return source
+    else:
+        raise ValueError(f"{source} is not a valid DataSource.")
```

Comment:
I think this can be replaced with `_to_datasource(source).value`

_to_datasource already raises ValueError so it's safe to use .value here.